dnd/config/spells.yaml

```diff
@@ -0,0 +1,1306 @@
+---
+name: Glyph
+sphere: arcane
+circle: 2
+range: "touch"
+area: "1 glyph"
+casting_time: "2 rounds"
+components: "SM"
+duration: 1 day or until discharged
+mechanics: special
+description: |
+  <p>This spell creates a magical glyph. The glyph must be written on a wood, paper or stone surface, and have a diameter of between 10 and 50 cm. It cannot be erased without magic or without destroying the surface it is written on. It consists of one rune, or a single sentence of no more than 6 words.</p>
+  <p>Targets within 2 meters, or targets within 5 meters looking at the rune, discharge it and activate its effects. If they try to avoid looking at it, they may save vs will to avoid the effect, but while doing so they face penalties as if blind, unless they have the blind-fighting skill.</p>
+  <p>The standard effect of the glyph is an explosion that inflicts 3d4 heat damage. The explosion affects everything in a 3 meter radius. A successful save vs reflex halves the damage.</p>
+  <p>The glyph is easily visible. It can be circumvented by a stealth-check with a -5 penalty. It can be safely removed by a lockpicking-check with a -5 penalty, but the character removing it must already be inside the radius.</p>
+  <p>Higher circle casters can add more effects to the spell, depending on their reserve.</p>
+  <table>
+  <tr><td>reserve</td><td>effect</td></tr>
+  <tr><td>1 circle</td><td>the damage of the glyph increases to 1d4 damage per caster level in a 5 meter radius</td></tr>
+  <tr><td>1 circle</td><td>the caster can set the damage type to acid, cold, lightning or sonic</td></tr>
+  <tr><td>1 circle</td><td>the closest target is paralysed for 1 turn / level, fortitude negates</td></tr>
+  <tr><td>1 circle</td><td>duration becomes permanent until discharged</td></tr>
+  <tr><td>1 circle</td><td>stealth-checks to pass the glyph face another -5 penalty (this can be taken multiple times)</td></tr>
+  <tr><td>1 circle</td><td>lockpicking-checks to remove the glyph face another -5 penalty (this can be taken multiple times)</td></tr>
+  <tr><td>1 circle</td><td>the glyph is transparent, it takes an observation-check to detect it</td></tr>
+  <tr><td>1 circle</td><td>the glyph hovers in mid-air</td></tr>
+  <tr><td>3 circles</td><td>the closest target turns to stone, permanently, fortitude negates</td></tr>
+  </table>
+  <p>If the DM allows, the caster can also add a second spell to the glyph as if it were an additional effect. This spell must be one that the caster has prepared, and it is expended (considered cast) when it is incorporated into the glyph. The circle of the second spell, as well as any effects it carries, are treated as if they are part of the reserve and extra effects of the glyph.</p>
+---
+name: Gorgon
+sphere: thought
+circle: 1
+range: "short"
+area: "1 creature"
+casting_time: "3"
+components: "VSM"
+duration:	5 rounds or until discharged
+mechanics:	will negates, twice
```

Comment:
Again, the extra whitespace. It shouldn't break anything, but I prefer conformity.